Live/LiveEventWithDVR/Program.cs

```diff
@@ -1,663 +1,562 @@
+string drvAssetFilterName = "filter-" + uniqueness;
+string streamingLocatorName = "streamingLocator" + uniqueness;
+string streamingEndpointName = "default"; // Change this to your specific streaming endpoint name if not using "default"
+bool stopEndpoint = false;
+
+// In this sample, we use Event Grid to listen to the notifications through an Azure Event Hub. 
+// If you do not provide an Event Hub config, the sample will fall back to polling the job for status. 
+// For production ready code, it is always recommended to use Event Grid instead of polling on the Job status. 
+
+EventProcessorClient processorClient = null;
+BlobContainerClient storageClient = null;
+MediaServicesEventProcessor mediaEventProcessor = null;
+
+try
 {
-    class Program
+    #region CreateLiveEvent
+    Console.WriteLine($"Creating a live event named {liveEventName}");
+    Console.WriteLine();
+
+    // Creating the LiveEvent - the primary object for live streaming in AMS. 
+    // See the overview - https://docs.microsoft.com/azure/media-services/latest/live-streaming-overview
+
+    // Create the LiveEvent
+
+    // Understand the concepts of what a live event and a live output is in AMS first!
+    // Read the following - https://docs.microsoft.com/azure/media-services/latest/live-events-outputs-concept
+    // 1) Understand the billing implications for the various states
+    // 2) Understand the different live event types, pass-through and encoding
+    // 3) Understand how to use long-running async operations 
+    // 4) Understand the available Standby mode and how it differs from the Running Mode. 
+    // 5) Understand the differences between a LiveOutput and the Asset that it records to.  They are two different concepts.
+    //    A live output can be considered as the "tape recorder" and the Asset is the tape that is inserted into it for recording.
+    // 6) Understand the advanced options such as low latency, and live transcription/captioning support. 
+    //    Live Transcription - https://docs.microsoft.com/en-us/azure/media-services/latest/live-transcription
+    //    Low Latency - https://docs.microsoft.com/en-us/azure/media-services/latest/live-event-latency
+
+    // When broadcasting to a live event, please use one of the verified on-premises live streaming encoders.
+    // While operating this tutorial, it is recommended to start out using OBS Studio before moving to another encoder. 
+
+    // Note: When creating a LiveEvent, you can specify allowed IP addresses in one of the following formats:                 
+    //      IpV4 address with 4 numbers
+    //      CIDR address range  
+
+    IPRange allAllowIpRange = new IPRange
     {
-        // Set this variable to true if you want to authenticate Interactively through the browser using your Azure user account
-        private const bool UseInteractiveAuth = false;
-
-        public static async Task Main(string[] args)
+        Name = "AllowAll",
```

Comment:
@jonpayne  - Sholud we add IPv6 Allow All here now?

Reply:
I nearly made a similar comment, but I didn't want to start using the PR to sneak other features into the samples.

To support any IPv4 or IPv6 address I would do this:
```csharp
    await mediaServicesAccount.GetLiveEvents().CreateOrUpdateAsync(WaitUntil.Completed, "a", new LiveEventData(AzureLocation.WestUS)
    {
        Input = new LiveEventInput(LiveEventInputProtocol.Rtmp)
        {
            IPAllowedIPs =
            {
                new IPRange { Name = "Allow all IPv4 addresses", Address = IPAddress.Any, SubnetPrefixLength = 0 },
                new IPRange { Name = "Allow all IPv6 addresses", Address = IPAddress.IPv6Any, SubnetPrefixLength = 0 },
            }
        }
    });

```